include/object_neo_device.h

```diff
@@ -42,7 +42,7 @@
 #define ICS_HANDLE_PY_TYPE T_INT
 #else
 #define ICS_HANDLE void*
-#define ICS_HANDLE_PY_TYPE T_INT
+#define ICS_HANDLE_PY_TYPE T_ULONGLONG
```

Comment:
Are there any changes other than this one which relate to 64-bit support?

Reply:
@grant-allan-ctct sorry I never responded to this, I thought I did... No other changes were needed here. I'm going to merge in soon.